validate/views.py

```diff
@@ -77,6 +78,10 @@ class V2ValidateYamlHandler(V1ValidateYamlHandler):
     permission_classes = [AllowAny]
 
     def post(self, request, *args, **kwargs):
+        source = self.request.query_params.get("source", None)
+        if source is not None:
+            sentry_metrics.set("validate_yaml_source", source)
```

Comment:
lmk if I'm doing this wrong - first time using sentry metrics

Reply:
I believe this would be using `incr`, double check here https://develop.sentry.dev/sdk/metrics/
something like:
```
source = self.request.query_params.get("source", "n/a")
sentry_metrics.incr("validate_v2", tags={"source": source})
```